python_modules/dagster-airflow/dagster_airflow/factory.py

```diff
@@ -26,8 +25,77 @@
 }
 
 
+def format_config_for_graphql(config):
```

Comment:
ooooo this would be a nice utility for graphql development.

Reply:
i am a little surprised this doesn't already exist somewhere